### DISCUSSION TITLE
MYSQL Settings

### DISCUSSION THREAD

**Discussion by unc1** [apache/skywalking#8140] [Q&A]
Hi,
Are there any recommended MYSQL DB settings that I should be using. I am having to tweak a lot of the default settings due to errors in the logs about DB issues (rows, columns, size of things, etc..) Thanks.

**Comment by wu-sheng** [comment]
You could refer how to run e2e test with MySQL storage, it nearly just simply uses 8.0.13. https://github.com/apache/skywalking/blob/master/test/e2e-v2/cases/storage/mysql/docker-compose.yml

Maybe you were using old version?
